pom.xml

```diff
@@ -328,48 +362,22 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.19.1</version>
+                <version>2.21.0</version>
                 <dependencies>
                     <dependency>
-                        <groupId>org.apache.maven.surefire</groupId>
-                        <artifactId>surefire-junit47</artifactId>
-                        <version>2.19</version>
+                        <groupId>org.junit.platform</groupId>
+                        <artifactId>junit-platform-surefire-provider</artifactId>
+                        <version>1.2.0</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.junit.jupiter</groupId>
+                        <artifactId>junit-jupiter-engine</artifactId>
+                        <version>${junit5Version}</version>
                     </dependency>
                 </dependencies>
                 <configuration>
-                    <!-- ${argLine} gets defined by jacoco plugin -->
-                    <argLine>${argLine} -Xmx1g</argLine>
```

Comment:
Do we no longer need the extra heap size?

Reply:
Does not appear to, I haven't had issues locally or on travis. 🤞